src/main/java/seedu/mark/logic/commands/results/GotoCommandResult.java

```diff
@@ -0,0 +1,33 @@
+package seedu.mark.logic.commands.results;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.mark.model.bookmark.Url;
+
+/**
+ * Represents the result of a goto command execution.
+ */
+public class GotoCommandResult extends CommandResult {
+    private Url url;
+
+    /**
+     * Constructs an {@code GotoCommandResult} with the feedback and url.
+     *
+     * @param feedbackToUser feedback
+     * @param url            url to be opened
```

Comment:
```suggestion
     * @param url The url to be opened
```
Perhaps we can capitalise the first character of the description for our `@param`, `@return` and `@throws` (standardises to the style they used in AB3)